apps/src/templates/manageStudents/ManageStudentsTable.jsx

```diff
@@ -312,11 +315,27 @@ class ManageStudentsTable extends Component {
       },
     ];
 
-    if (loginType === SectionLoginType.word || loginType === SectionLoginType.picture || loginType === SectionLoginType.email) {
-      return dataColumns.concat(controlsColumns);
-    } else {
-      return dataColumns;
+    const allowedPasswordColumn = [
+      SectionLoginType.word,
+      SectionLoginType.picture,
+      SectionLoginType.email,
+    ];
+    const allowedControlsColumn = [
+      SectionLoginType.word,
+      SectionLoginType.picture,
+      SectionLoginType.email,
+      SectionLoginType.google_classroom,
+      SectionLoginType.clever,
+    ];
```

Comment:
Nit: I'd define these static constants outside the `getColumns` method - probably in module scope.  Could also name them something more obvious, like `LOGIN_TYPES_WITH_PASSWORD_COLUMN`.